sdk/vc-data/tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "moduleResolution": "node",
     "jsx": "react",
     "esModuleInterop": true,
-    "outDir": "dist"
+    "outDir": "dist",
+    "skipLibCheck": true
```

Comment:
@oleksiipiliugin why we add skipLibCheck?

Reply:
pls see [docs](https://www.typescriptlang.org/tsconfig#skipLibCheck)